files/zh-cn/web/media/audio_and_video_delivery/buffering_seeking_time_ranges/index.md

```diff
@@ -11,7 +11,7 @@ slug: Web/Media/Audio_and_video_delivery/buffering_seeking_time_ranges
 
 `Buffered` 属性会告诉我们媒体的哪一部分已经下载好了。它返回一个 {{ domxref("TimeRanges") }} 对象，表名哪些块已经下载。这通常是连续的但是如果用户在缓冲时跳过，就可能会有缺口。
 
-它与 {{htmlelement("audio") }} 或 {{htmlelement("video") }}一起工作; 现在我们来考虑一个简单的 aodio 例子：
+它与 {{htmlelement("audio") }} 或 {{htmlelement("video") }}一起工作; 现在我们来考虑一个简单的 audio 例子：
```

Comment:
Let's translate the word here since it's not a 'proper noun'.

```suggestion
它与 {{htmlelement("audio") }} 或 {{htmlelement("video") }} 一起工作，现在我们来考虑一个简单的音频示例：
```